main.go

```diff
@@ -354,15 +368,17 @@ func newUser(s ssh.Session) *User {
 		Log.Println("Could not load user:", err)
 	}
 
-	if len(Backlog) > 0 {
-		lastStamp := Backlog[0].timestamp
-		u.rWriteln(fmtTime(u, lastStamp))
-		for i := range Backlog {
-			if Backlog[i].timestamp.Sub(lastStamp) > time.Minute {
-				lastStamp = Backlog[i].timestamp
-				u.rWriteln(fmtTime(u, lastStamp))
+	if !Config.Private {
```

Comment:
we've already refused non-allowed people, right? we can show the allowed people the backlog

Reply:
If people need private chats, they might need it to discuss sensible manner. In that case, they probably wouldn't want some people reading their writings in the future without them being able to prevent that.

Reply:
okay let's add a comment that says this ^

Reply:
Done!